File: 1-two-sum/1-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int>mp;
        for(int i=0;;i++){
            auto it=mp.find(target-nums[i]);
            if(it!=mp.end())
            return vector<int>{i,it->second};
            
            mp[nums[i]]=i;
        }
    }
};

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, n: List[int], t: int) -> List[int]:
        for i in range(0,len(n)):
            if t-n[i] in n[i+1:]:
                return [i,n[i+1:].index(t-n[i])+i+1]
        